config/runtime.exs

```diff
@@ -6,17 +6,9 @@ import Config
 
 eb_env_name = System.get_env("ENVIRONMENT_NAME")
 
-api_v3_url =
-  case eb_env_name do
-    "screens-prod" -> "https://api-v3.mbta.com/"
-    "screens-dev" -> "https://api-dev.mbtace.com/"
-    "screens-dev-green" -> "https://api-dev-green.mbtace.com/"
-    _ -> System.get_env("API_V3_URL", "https://api-v3.mbta.com/")
-  end
-
 unless config_env() == :test do
   config :screens,
-    api_v3_url: api_v3_url,
+    api_v3_url: System.get_env("API_V3_URL") || raise("Must set API_V3_URL"),
```

Comment:
`System.fetch_env!/1` does the equivalent of this. (oops, commented this on the commit itself before, which was then rewritten out of existence)

Reply:
yep i can do that and keep the default value. the evolution of this change is that `api_v3_url` was originally calculated outside of this `unless` so i removed the default to make it harder to accidentally run tests against they actual V3 API.